src/main/java/seedu/address/logic/parser/EditAppointmentCommandParser.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DOCTOR;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PATIENT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIMESLOT_DURATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIMESLOT_END;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIMESLOT_START;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.EditAppointmentCommand;
+import seedu.address.logic.commands.EditAppointmentCommand.EditAppointmentDescriptor;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+
+public class EditAppointmentCommandParser implements Parser<EditAppointmentCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the EditCommand
+     * and returns an EditCommand object for execution.
```

Comment:
docs to update to EditAppointmentCommand.

Reply:
updated. Thnanks